pkg/maps/ctmap/ctmap.go

```diff
@@ -544,17 +553,17 @@ func PurgeOrphanNATEntries(ctMapTCP, ctMapAny *Map) *NatGCStats {
 		} else {
 			natMap = natm
 		}
-	}
 
-	if natMap == nil {
-		return nil
+		if err := natMap.Open(); err != nil {
```

Comment:
`natMap` can be `nil` here - move it into the `else` case (or early return in case we're not able to get a per-cluster map)